docs/elements.md

```diff
@@ -1906,7 +1906,7 @@ Displays boilerplate HTML from templates in a reusable way.
 #### Sample element
 
 ```html
-<pl-template file-name="outer_template.mustache" subdirectory="templates">
+<pl-template file-name="templates/outer_template.mustache" directory="serverFilesCourse">
```

Comment:
`serverFilesCourse` is the default directory, maybe remove the attribute?

Reply:
Good catch! Going along with what you mentioned in your comment, I'll update this example with a template file that has a short example of using the uuid key.

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/8434/commits/1bc546cef6db9bad95776c83a59fc83da54d36ad